README.rst

```diff
@@ -15,104 +15,171 @@ sqlitedict -- persistent ``dict``, backed-up by SQLite and pickle
 A lightweight wrapper around Python's sqlite3 database with a simple, Pythonic
 dict-like interface and support for multi-thread access:
 
+Usage
+=====
+
+Write
+-----
+
 .. code-block:: python
 
-  >>> from sqlitedict import SqliteDict
-  >>> mydict = SqliteDict('./my_db.sqlite', autocommit=True)
-  >>> mydict['some_key'] = any_picklable_object
-  >>> print mydict['some_key']  # prints the new value
-  >>> for key, value in mydict.iteritems():
-  >>>     print key, value
-  >>> print len(mydict) # etc... all dict functions work
-  >>> mydict.close()
+    from sqlitedict import SqliteDict
```

Comment:
Please convert these to doctest-compatible examples and make sure that they pass.

https://docs.python.org/3.8/library/doctest.html

Reply:
Oh interesting. The ">>>" at the beginning of every line is a thing for doctest?
They prevent to copy/paste the examples, so they were bad formatting to fix IMO.

Reply:
Yes, they were intentional. Please see the doctest documentation for more.

For copypasting, some Python shells (e.g. ipython) automatically strip the >>> in incoming text.